src/evaluation.py

```diff
@@ -0,0 +1,598 @@
+import ast
```

Comment:
A couple of articles that might be useful for refactoring this file:
https://dzone.com/articles/the-stepdown-rule
https://dzone.com/articles/slap-your-methods-and-dont-make-me-think

Reply:
Will do via PR#3